mdsf-vscode/README.md

```diff
@@ -0,0 +1,71 @@
+# mdsf-vscode README
```

Comment:
The content of this file is a total placeholder.

Maybe you missed it

Reply:
Moved to #649